ktbridge/plugin/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+plugins {
+  kotlin("jvm")
+  kotlin("kapt")
+  id("com.github.gmazzo.buildconfig")
+}
+
+dependencies {
+  compileOnly("org.jetbrains.kotlin:kotlin-compiler-embeddable")
```

Comment:
This module should also have a `gradle.properties` with:
```properties
# We get the stdlib transitively as a compileOnly dependency.
kotlin.stdlib.default.dependency=false
```

Reply:
Ooooh I like that.